internal/output/opentelemetry/output.go

```diff
@@ -173,19 +179,57 @@ func (o *Output) dispatch(entry metrics.Sample) error {
 
 		trend.Record(ctx, entry.Value, attributeSetOpt)
 	case metrics.Rate:
-		nonZero, total, err := o.metricsRegistry.getOrCreateCountersForRate(name)
+		var err error
+		if o.config.SingleCounterForRate.Bool {
+			err = o.singleCounterForRate(ctx, name, attributeSetOpt, entry)
+		} else {
+			// Deprecated path, remove on k6 1.4.0
```

Comment:
Kind reminder for creating an issue for this, and assigning it to the 1.4.0 milestone.

Reply:
```suggestion
			// Deprecated path, remove with https://github.com/grafana/k6/issues/5185 
```

I do not think it will be in v1.4.0 as we will be releasing this change there. I am also not certain if single version is good idea. Especially as maintaining this seems to not be particularly problematic. So I would probably leave it without specific version and potentially remove it in v1.5.0 if there is no feedback against it.